packages/web/src/components/user-generated-text/UserGeneratedTextV2.tsx

```diff
@@ -111,9 +112,11 @@ const HandleLink = ({
   })
 
   return user ? (
-    <TextLink {...other} to={profilePage(user.handle)}>
-      {handle}
-    </TextLink>
+    <ArtistPopover handle={user.handle} component='span'>
+      <TextLink {...other} to={profilePage(user.handle)}>
```

Comment:
Do you still need text link here? Or does the popover component handle it?

Reply:
i think the popover is purely just the popover, so it's needed